### PR TITLE
PP-6688 Remove `check your answers` pages

### PR DESCRIPTION
**Pull request by kbottla**
## WHAT
- Removed `Check your answers` pages for all onboarding tasks. Data is submitted to Stripe as soon as form is submitted.
- Renamed button 'Continue' to 'Save and continue` to reflect the action
- Splits VAT number and Company number to individual tasks and data is submitted to Stripe individually.
     - Defaults `Company number` to `NOTAPPLI` as Stripe needs some data (which is being addressed by Stripe) even when company number is not available.
- Removed loads of cypress tests for combined `vat_number_company_number` task as they are no longer needed.

### CODE REVIEW

File: app/controllers/stripe-setup/vat-number-company-number/company-number/get.controller.js [hunk 1]
```diff
@@ -1,27 +1,8 @@
...
-  } else {
-    pageData = {
-      errors: {},
-      companyNumberDeclaration: '',
-      companyNumber: ''
-    }
-  }
-
-  return response(req, res, 'stripe-setup/vat-number-company-number/company-number', pageData)
+  return response(req, res, 'stripe-setup/vat-number-company-number/company-number')
```
Comment (iqbalgds) [inline app/controllers/stripe-setup/vat-number-company-number/company-number/get.controller.js:7]: Minor point - Seeing as we are splitting VAT and Company number - is it worth changing the URL to: 
`stripe-setup/company-number`?

**Comment by iqbalgds** [comment]
Amazing work - left 1 minor point.  Everything else looks good to me.
As its such a big PR - will let someone else check it as well.

File: app/controllers/stripe-setup/vat-number-company-number/company-number/get.controller.js (hunk 1 above)
Comment (kbottla) [reply app/controllers/stripe-setup/vat-number-company-number/company-number/get.controller.js:7]: yes, planning to change it separately as updating here could make the PR big

File: app/models/StripeAccountSetup.class.js [hunk 2]
```diff
@@ -5,6 +5,8 @@ class StripeAccountSetup {
     this.bankAccount = opts.bank_account
     this.vatNumberCompanyNumber = opts.vat_number_company_number
```
Comment (sfount) [inline app/models/StripeAccountSetup.class.js:6]: I think uses of this have been removed so it can probably be removed from the model

Comment (kbottla) [reply app/models/StripeAccountSetup.class.js:6]: 👍 this is going into separate PR to remove here and all references

**Pull request merged**